spec/factories/bookings.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+  factory :booking do
+    name { "test user" }
+    email { "testuser1@mail.com" }
+    mobile { "9999999999" }
+    booking_date { "2021/12/20" }
+    from_time { "12:00:00" }
+    to_time { "14:00:00" }
+
+    factory :booked do
```

Comment:
`booked` won't give complete idea about what we are going to create. Please rename it to `booked_booking`.

Reply:
Changes are done.